### DISCUSSION TITLE
How to use React-Markdown to show streaming response from OpenAI

### DISCUSSION THREAD

**Discussion by sanket20138339** [remarkjs/remark#1342] [Q&A] [answered]
I am using React-Markdown and it works fine to show response from Open AI.
But I handle streaming response it shows response in not correct format, my response is coming as a markdown.

Can you please help me here how to use it to display the markdown in streaming response

**Comment by remcohaszing** [comment, marked as answer]
See https://github.com/orgs/remarkjs/discussions/1332 (the whole thread)
